metrics/integration_test.go

```diff
@@ -227,12 +227,6 @@ var _ = Describe("Metrics Integration", Ordered, func() {
 	})
 
 	Context("When RegisterInvalidSnapshot", func() {
-		It("increments the 'SnapshotInvalidTotal' metric", func() {
-			for i := 0; i < 10; i++ {
-				RegisterInvalidSnapshot("AppStudioIntegrationStatus", "invalid")
```

Comment:
Tests are failing, we need to move this with for cycle into next test, we have to call `RegisterInvalidSnapshot` 10 times

Reply:
Yep, I moved it to draft when it failed the tests as I had to go to meetings etc. It will be tomorrow before i get back to it. thanks for the tip, will save me time :)